[2024-06-14T12:00:11.381+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T12:00:02.338214+00:00 [queued]>
[2024-06-14T12:00:11.392+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T12:00:02.338214+00:00 [queued]>
[2024-06-14T12:00:11.393+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-06-14T12:00:11.410+0000] {taskinstance.py:1380} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-06-14 12:00:02.338214+00:00
[2024-06-14T12:00:11.437+0000] {standard_task_runner.py:57} INFO - Started process 10260 to run task
[2024-06-14T12:00:11.449+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_spark_dag', 'spark_submit_task', 'manual__2024-06-14T12:00:02.338214+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/simple_spark_dag.py', '--cfg-path', '/tmp/tmphusqaz72']
[2024-06-14T12:00:11.462+0000] {standard_task_runner.py:85} INFO - Job 70: Subtask spark_submit_task
[2024-06-14T12:00:11.591+0000] {task_command.py:415} INFO - Running <TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T12:00:02.338214+00:00 [running]> on host 963b3fbfa4c9
[2024-06-14T12:00:11.670+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple_spark_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-14T12:00:02.338214+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-14T12:00:02.338214+00:00'
[2024-06-14T12:00:11.693+0000] {base.py:73} INFO - Using connection ID 'Spark_connection' for task execution.
[2024-06-14T12:00:11.696+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py
[2024-06-14T12:00:12.868+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.9/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-06-14T12:00:26.092+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO SparkContext: Running Spark version 3.4.0
[2024-06-14T12:00:26.471+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-06-14T12:00:26.790+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceUtils: ==============================================================
[2024-06-14T12:00:26.793+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-06-14T12:00:26.796+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceUtils: ==============================================================
[2024-06-14T12:00:26.798+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO SparkContext: Submitted application: Spark MinIO Example
[2024-06-14T12:00:26.888+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-06-14T12:00:26.991+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceProfile: Limiting resource is cpu
[2024-06-14T12:00:26.996+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:26 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-06-14T12:00:27.214+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SecurityManager: Changing view acls to: ***
[2024-06-14T12:00:27.220+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SecurityManager: Changing modify acls to: ***
[2024-06-14T12:00:27.221+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SecurityManager: Changing view acls groups to:
[2024-06-14T12:00:27.223+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SecurityManager: Changing modify acls groups to:
[2024-06-14T12:00:27.224+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-06-14T12:00:27.768+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO Utils: Successfully started service 'sparkDriver' on port 40473.
[2024-06-14T12:00:27.835+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SparkEnv: Registering MapOutputTracker
[2024-06-14T12:00:27.930+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO SparkEnv: Registering BlockManagerMaster
[2024-06-14T12:00:27.984+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-06-14T12:00:27.989+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:27 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-06-14T12:00:28.011+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-06-14T12:00:28.126+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-e34261ab-e6ea-4d22-9a09-14fcacf0944a
[2024-06-14T12:00:28.195+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-06-14T12:00:28.259+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-06-14T12:00:28.505+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-06-14T12:00:28.585+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-06-14T12:00:28.813+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-06-14T12:00:28.957+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:28 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.3:7077 after 42 ms (0 ms spent in bootstraps)
[2024-06-14T12:00:31.316+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20240614120031-0009
[2024-06-14T12:00:31.355+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 38291.
[2024-06-14T12:00:31.362+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO NettyBlockTransferService: Server created on 963b3fbfa4c9:38291
[2024-06-14T12:00:31.366+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-06-14T12:00:31.376+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20240614120031-0009/0 on worker-20240614074744-172.18.0.5-36477 (172.18.0.5:36477) with 2 core(s)
[2024-06-14T12:00:31.382+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 963b3fbfa4c9, 38291, None)
[2024-06-14T12:00:31.387+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO StandaloneSchedulerBackend: Granted executor ID app-20240614120031-0009/0 on hostPort 172.18.0.5:36477 with 2 core(s), 1024.0 MiB RAM
[2024-06-14T12:00:31.391+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO BlockManagerMasterEndpoint: Registering block manager 963b3fbfa4c9:38291 with 434.4 MiB RAM, BlockManagerId(driver, 963b3fbfa4c9, 38291, None)
[2024-06-14T12:00:31.393+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 963b3fbfa4c9, 38291, None)
[2024-06-14T12:00:31.397+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:31 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 963b3fbfa4c9, 38291, None)
[2024-06-14T12:00:32.027+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:32 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-06-14T12:00:32.482+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:32 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-06-14T12:00:32.496+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:32 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-06-14T12:00:36.867+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:36 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20240614120031-0009/0 is now RUNNING
[2024-06-14T12:00:40.360+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:40 WARN FileStreamSink: Assume no metadata directory. Error while looking for metadata directory in the path: s3a://data-lake/vnindex.csv.
[2024-06-14T12:00:40.395+0000] {spark_submit.py:571} INFO - java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T12:00:40.398+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2688)
[2024-06-14T12:00:40.399+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.getFileSystemClass(FileSystem.java:3431)
[2024-06-14T12:00:40.404+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3466)
[2024-06-14T12:00:40.410+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.access$300(FileSystem.java:174)
[2024-06-14T12:00:40.411+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:3574)
[2024-06-14T12:00:40.413+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:3521)
[2024-06-14T12:00:40.416+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:540)
[2024-06-14T12:00:40.423+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Path.getFileSystem(Path.java:365)
[2024-06-14T12:00:40.424+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.streaming.FileStreamSink$.hasMetadata(FileStreamSink.scala:53)
[2024-06-14T12:00:40.425+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:366)
[2024-06-14T12:00:40.426+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-06-14T12:00:40.427+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-06-14T12:00:40.428+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-06-14T12:00:40.430+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-06-14T12:00:40.431+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-06-14T12:00:40.435+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-06-14T12:00:40.437+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-06-14T12:00:40.448+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-06-14T12:00:40.450+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-06-14T12:00:40.452+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-06-14T12:00:40.461+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-06-14T12:00:40.468+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-06-14T12:00:40.496+0000] {spark_submit.py:571} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-06-14T12:00:40.514+0000] {spark_submit.py:571} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-06-14T12:00:40.515+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-06-14T12:00:40.516+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-06-14T12:00:40.516+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-06-14T12:00:40.521+0000] {spark_submit.py:571} INFO - Caused by: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T12:00:40.522+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClassByName(Configuration.java:2592)
[2024-06-14T12:00:40.522+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2686)
[2024-06-14T12:00:40.528+0000] {spark_submit.py:571} INFO - ... 26 more
[2024-06-14T12:00:40.535+0000] {spark_submit.py:571} INFO - Traceback (most recent call last):
[2024-06-14T12:00:40.536+0000] {spark_submit.py:571} INFO - File "/opt/***/jobs/python/test2.py", line 10, in <module>
[2024-06-14T12:00:40.561+0000] {spark_submit.py:571} INFO - df = spark.read.csv(minio_url, header=True, inferSchema=True)
[2024-06-14T12:00:40.562+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 727, in csv
[2024-06-14T12:00:40.606+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-06-14T12:00:40.608+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 169, in deco
[2024-06-14T12:00:40.610+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-06-14T12:00:40.627+0000] {spark_submit.py:571} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o30.csv.
[2024-06-14T12:00:40.633+0000] {spark_submit.py:571} INFO - : java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T12:00:40.640+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2688)
[2024-06-14T12:00:40.647+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.getFileSystemClass(FileSystem.java:3431)
[2024-06-14T12:00:40.667+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3466)
[2024-06-14T12:00:40.719+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.access$300(FileSystem.java:174)
[2024-06-14T12:00:40.740+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:3574)
[2024-06-14T12:00:40.760+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:3521)
[2024-06-14T12:00:40.772+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:540)
[2024-06-14T12:00:40.786+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Path.getFileSystem(Path.java:365)
[2024-06-14T12:00:40.800+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$1(DataSource.scala:724)
[2024-06-14T12:00:40.815+0000] {spark_submit.py:571} INFO - at scala.collection.immutable.List.map(List.scala:293)
[2024-06-14T12:00:40.829+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.checkAndGlobPathIfNecessary(DataSource.scala:722)
[2024-06-14T12:00:40.862+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.checkAndGlobPathIfNecessary(DataSource.scala:551)
[2024-06-14T12:00:40.869+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:404)
[2024-06-14T12:00:40.876+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-06-14T12:00:40.889+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-06-14T12:00:40.897+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-06-14T12:00:40.910+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-06-14T12:00:40.917+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-06-14T12:00:40.931+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-06-14T12:00:40.942+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-06-14T12:00:40.944+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-06-14T12:00:40.947+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-06-14T12:00:40.948+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-06-14T12:00:40.952+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-06-14T12:00:40.956+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-06-14T12:00:40.958+0000] {spark_submit.py:571} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-06-14T12:00:40.961+0000] {spark_submit.py:571} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-06-14T12:00:40.962+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-06-14T12:00:40.967+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-06-14T12:00:40.972+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-06-14T12:00:40.980+0000] {spark_submit.py:571} INFO - Caused by: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T12:00:40.985+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClassByName(Configuration.java:2592)
[2024-06-14T12:00:40.991+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2686)
[2024-06-14T12:00:40.995+0000] {spark_submit.py:571} INFO - ... 29 more
[2024-06-14T12:00:41.000+0000] {spark_submit.py:571} INFO - 
[2024-06-14T12:00:44.586+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:44 INFO SparkContext: Invoking stop() from shutdown hook
[2024-06-14T12:00:44.616+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:44 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-06-14T12:00:45.171+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:45 INFO SparkUI: Stopped Spark web UI at http://963b3fbfa4c9:4040
[2024-06-14T12:00:45.349+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:45 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-06-14T12:00:45.880+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:45 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-06-14T12:00:46.058+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-06-14T12:00:46.188+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO MemoryStore: MemoryStore cleared
[2024-06-14T12:00:46.198+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO BlockManager: BlockManager stopped
[2024-06-14T12:00:46.215+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-06-14T12:00:46.224+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-06-14T12:00:46.304+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO SparkContext: Successfully stopped SparkContext
[2024-06-14T12:00:46.306+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO ShutdownHookManager: Shutdown hook called
[2024-06-14T12:00:46.308+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO ShutdownHookManager: Deleting directory /tmp/spark-f1f4178f-a022-44b3-a1d6-f4c481d94422
[2024-06-14T12:00:46.337+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO ShutdownHookManager: Deleting directory /tmp/spark-588c88f4-e689-42fa-8018-71bc7bf568ce
[2024-06-14T12:00:46.368+0000] {spark_submit.py:571} INFO - 24/06/14 12:00:46 INFO ShutdownHookManager: Deleting directory /tmp/spark-588c88f4-e689-42fa-8018-71bc7bf568ce/pyspark-09ba6115-1673-4daf-8744-487797a31fa7
[2024-06-14T12:00:46.802+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py. Error code is: 1.
[2024-06-14T12:00:46.858+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=simple_spark_dag, task_id=spark_submit_task, execution_date=20240614T120002, start_date=20240614T120011, end_date=20240614T120046
[2024-06-14T12:00:46.935+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 70 for task spark_submit_task (Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py. Error code is: 1.; 10260)
[2024-06-14T12:00:47.030+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-14T12:00:47.227+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
